apps/mobile-wallet/src/contexts/walletConnect/WalletConnectContext.tsx

```diff
@@ -334,7 +334,8 @@ export const WalletConnectContextProvider = ({ children }: { children: ReactNode
             autoHide: false
           })
         } else {
-          showExceptionToast(e, 'Could not build transaction')
+          if (!['alph_requestNodeApi', 'alph_requestExplorerApi'].includes(requestEvent.params.request.method))
+            showExceptionToast(e, 'Could not build transaction')
```

Comment:
There is no point showing this error to the user. The user only cares about errors regarding signing.